Failure in System.Diagnostics.Tests.ActivityTests.DiagnosticSourceStartStop

@joperezr can you find the right owner for DiagnosticSource? This gave us a red badge this morning. This test may need some logging adding, and leave enabled.

https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_win7_release/lastCompletedBuild/consoleText

```
  xUnit.net console test runner (64-bit .NET Core)
  Copyright (C) 2014 Outercurve Foundation.
  
  Discovering: System.Diagnostics.DiagnosticSource.Tests
  Discovered:  System.Diagnostics.DiagnosticSource.Tests
  Starting:    System.Diagnostics.DiagnosticSource.Tests
     System.Diagnostics.Tests.ActivityTests.DiagnosticSourceStartStop [FAIL]
        Assert.True() Failure
        Expected: True
        Actual:   False
        Stack Trace:
           D:\j\workspace\outerloop_win---75d58f69\src\System.Diagnostics.DiagnosticSource\tests\ActivityTests.cs(378,0): at System.Diagnostics.Tests.ActivityTests.DiagnosticSourceStartStop()
  Finished:    System.Diagnostics.DiagnosticSource.Tests
  ```

Failed in NetFX tests for #21739

https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/616/consoleFull

```
16:52:03 D:\j\workspace\windows-TGrou---2a8f9c29\Tools\tests.targets(346,5): warning MSB3073: The command "D:\j\workspace\windows-TGrou---2a8f9c29\bin/Windows_NT.AnyCPU.Release/System.Diagnostics.DiagnosticSource.Tests/netfx//RunTests.cmd D:\j\workspace\windows-TGrou---2a8f9c29\bin/testhost/netfx-Windows_NT-Release-x86/" exited with code 1. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Diagnostics.DiagnosticSource\tests\System.Diagnostics.DiagnosticSource.Tests.csproj]
16:52:03 D:\j\workspace\windows-TGrou---2a8f9c29\Tools\tests.targets(354,5): error : One or more tests failed while running tests from 'System.Diagnostics.DiagnosticSource.Tests' please check D:\j\workspace\windows-TGrou---2a8f9c29\bin/Windows_NT.AnyCPU.Release/System.Diagnostics.DiagnosticSource.Tests/netfx/testResults.xml for details! [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Diagnostics.DiagnosticSource\tests\System.Diagnostics.DiagnosticSource.Tests.csproj]
16:52:03 Command execution failed with exit code 1.


16:39:44   Running tests... Start time: 16:39:43.87
16:39:44   xUnit.net Console Runner (64-bit Desktop .NET 4.0.30319.42000)
16:39:46     Discovering: System.Diagnostics.DiagnosticSource.Tests
16:39:46   xUnit.net Console Runner (64-bit Desktop .NET 4.0.30319.42000)
16:39:46     Discovered:  System.Diagnostics.DiagnosticSource.Tests
16:39:46     Starting:    System.Diagnostics.DiagnosticSource.Tests
16:39:46     Discovering: System.Diagnostics.StackTrace.Tests
16:39:47     Discovered:  System.Diagnostics.StackTrace.Tests
16:39:47     Starting:    System.Diagnostics.StackTrace.Tests
16:39:47       System.Diagnostics.Tests.ActivityTests.StopWithoutTimestamp [FAIL]
16:39:47         Assert.InRange() Failure
16:39:47         Range:  (980 - 1.79769313486232E+308)
16:39:47         Actual: 972.2623
16:39:47         Stack Trace:
16:39:47           D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Diagnostics.DiagnosticSource\tests\ActivityTests.cs(327,0): at System.Diagnostics.Tests.ActivityTests.StopWithoutTimestamp()

```

--- a/src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs
+++ b/src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs
@@ -320,10 +320,11 @@ namespace System.Diagnostics.Tests
             // DateTime.UtcNow is not precise on some platforms, but Activity stop time is precise
             // in this test we set start time, but not stop time and check duration.
             //
-            // Let's check that duration is 1sec - maximum DateTime.UtcNow error or bigger.
+            // Let's check that duration is 1sec - 2 * maximum DateTime.UtcNow error or bigger.
+            // As both start and stop timestamps may have error.
             // There is another test (ActivityDateTimeTests.StartStopReturnsPreciseDuration) 
             // that checks duration precision on netfx.
-            Assert.InRange(activity.Duration.TotalMilliseconds, 1000 - MaxClockErrorMSec, double.MaxValue);
+            Assert.InRange(activity.Duration.TotalMilliseconds, 1000 - 2 * MaxClockErrorMSec, double.MaxValue);
         }
 
         /// <summary>

